indigo/distribution.yaml

```diff
@@ -9442,6 +9442,15 @@ repositories:
       url: https://github.com/So-Cool/report_card.git
       version: master
     status: maintained
+  repository_name:
```

Comment:
Please use a reasonable name here, e.g. `gauges`.

Once you have updated the above name you will have to remove the following part (since the package name now equals the repo name which doesn't need to explicitly stated):

```
packages:
- gauges
```